spec/rails_helper.rb

```diff
@@ -19,8 +19,7 @@
 # of increasing the boot-up time by auto-requiring all files in the support
 # directory. Alternatively, in the individual `*_spec.rb` files, manually
 # require only the support files necessary.
-#
-# Dir[Rails.root.join('spec', 'support', '**', '*.rb')].each { |f| require f }
+Dir[Rails.root.join('spec', 'support', '**', '*.rb')].each { |f| require f }
```

Comment:
I think this line won't be necessary when the code from #14 and its cleanup of the rspec configuration enters `master`.

Reply:
👍